assets/js/script.js

```diff
@@ -5,58 +5,68 @@ const formTitle = document.getElementById('title');
 const formAuthor = document.getElementById('author');
 const bookList = document.querySelector('#list');
 
-function addBook(title, author) {
-  if (title !== '' && author !== '') {
-    const book = {
-      id: books.length + 1,
-      title,
-      author,
-    };
-
-    books.push(book);
+class Book {
+  constructor() {
+    this.title = document.getElementById('title');
+    this.author = document.getElementById('author');
+  }
+
+  addBook(title, author) {
+    if (title !== '' && author !== '') {
+      const book = {
+        id: books.length + 1,
+        title,
+        author,
+      };
+
+      books.push(book);
```

Comment:
# Optional:
- Hi, nice job so far, I would advise you to use a method that doesn't mutate the array, therefore, use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push) method